src/styles/components/artifacts-section/styles.less

```diff
@@ -0,0 +1,3 @@
+.artifacts-section {
+  margin-top: 18px;
```

Comment:
I made some changes...
* I used the spacing variable and its screen sizes here
* I moved the styles to `src/styles/views/create-service-modal/`
* I added the TODO for creating a `FormSection` component